azavea/raster-foundry
# Allow deleting buckets linked to scenes

## Overview

Previously buckets that had scenes in them could not be deleted without first emptying all the scenes. With this `DELETE CASCADE` constraint, now the buckets can be deleted in one step.

## Testing Instructions

Check out this branch and run the migrations. Then,

 * Using either `cURL` of `HTTPie` or the Raster Foundry UI, create a bucket
 * Assign scenes to the bucket
 * Delete a scene without removing it from the bucket
 * The scene should be deleted
 * Delete the bucket without first removing the scenes
 * The bucket should be deleted

Connects #536
